### ISSUE TITLE
Program crashes when switching to eraser while drawing

### ISSUE THREAD

**Issue by lachy874125** [mbrlabs/Lorien#250]
**Lorien version:**
v0.5.0

**OS/device including version:**
Windows 10
Ubuntu 22.04

**Issue description:**
If I switch to the eraser while drawing with the brush or shape tools and then I release and click again the program crashes.

**Steps to reproduce:**

1. Select brush tool (also works with shape tools)
2. Start drawing (press left click and move the cursor)
3. While still holding down left click, switch to the eraser by clicking 'E' on the keyboard
4. Release left click
5. Press left click
6. Program crashes

**Comment by animatorkris** [comment]
Lorien version:
v0.5.0

OS/device including version:
Windows 10

Issue description:
I was able to reproduce this on a Huion tablet where one of the Pen's utility buttons was mapped to erase and I bumped it by accident while drawing. The app just evaporates. 

Steps to reproduce:

    Select brush tool
    Start drawing with the pen
    While drawing a line, click the bumper button mapped to eraser
    Release the bumper button
    The program crashes

**Comment by mbrlabs** [comment]
Thanks for reporting! This has already been fixed in the latest dev version. I will release v0.6.0 which will include this fix soon.

**Comment by 0xNF** [comment]
This issue isn't fixed as of the latest v0.6.0 release.

Here's some dxdiag info:

```txt
         Operating System: Windows 11 Pro 64-bit (10.0, Build 22631) (22621.ni_release.220506-1250)
                 Language: English (Regional Setting: English)
      System Manufacturer: Microsoft Corporation
             System Model: Surface Laptop Studio
                     BIOS: 23.200.143 (type: UEFI)
                Processor: 11th Gen Intel(R) Core(TM) i7-11370H @ 3.30GHz (8 CPUs), ~3.3GHz
                   Memory: 32768MB RAM
      Available OS Memory: 32602MB RAM
                Page File: 46084MB used, 11093MB available
              Windows Dir: C:\WINDOWS
          DirectX Version: DirectX 12
      DX Setup Parameters: Not found
         User DPI Setting: 144 DPI (150 percent)
       System DPI Setting: 144 DPI (150 percent)
          DWM DPI Scaling: UnKnown
                 Miracast: Available, with HDCP
Microsoft Graphics Hybrid: Supported
 DirectX Database Version: 1.4.7
           DxDiag Version: 10.00.22621.0001 64bit Unicode
```
